src/App.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import Router from './components/router'
+import Router from './components/Router'
```

Comment:
can we leave it lowercase?

Reply:
my vscode won't let me run it if i have it lowercase

Reply:
what error does it show?